functional-tests/envs/testenv.py

```diff
@@ -383,6 +384,225 @@ def init(self, ctx: flexitest.EnvContext) -> flexitest.LiveEnv:
         return BasicLiveEnv(svcs, bridge_pk, rollup_cfg)
 
 
+class DualSequencerMixedPolicyEnvConfig(flexitest.EnvConfig):
```

Comment:
Please add docstrings?
What is dual sequencer for example? And why do we need that?